faqs.html

```diff
@@ -5,29 +5,42 @@
   <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
   <ul class="collapsible black">
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <span class="card-title clem"><i class="material-icons">error_outline</i></span>
+        <p> As passionate as we are about cybersecurity, safety is our number one priority. We sincerely hoped that the situation around COVID-19 would improve enough to continue our event on Clemson University’s campus. However, based on local health experts and community leader recommendations, <strong>it has been determined that CUCTF 2020 will be a completely virtual event on October 3rd</strong>. We refuse to be setback from this announcement and will use this opportunity to:</p>
+          <ul class="browser-default">
+            <li>Allow more universities to participate by expanding our CTF infrastructure </li>
+            <li>Present opening and closing ceremonies via Zoom</li>
+            <li>Utilize Discord to answer questions, push notifications, and maintain a connection with participants</li>
+          </ul>
+      </div>
+    </li>
     <li>
       <div class="collapsible-header black clemson-orange-text">Where is the event being held?</div>
-      <div class="collapsible-body"><span>The even is being held at Clemson University's Watt Family Innovation Center. The address is <a rel="external noopener" href="https://goo.gl/maps/XZTw4u9Mu72V6P7s6" target="_blank">405 S Palmetto Blvd, Clemson, SC 29634</a>.</span></div>
+      <div class="collapsible-body"><span>CUCTF 2020 will be held online. More information on how to compete remotely will be announced closer to the competition start date. </span></div>
     </li>
     <li>
       <div class="collapsible-header black clemson-orange-text">When is the event?</div>
-      <div class="collapsible-body"><span>CUCTF is taking place October 3rd, 2020.</span></div>
+      <div class="collapsible-body"><span>CUCTF 2020 will take place October 3rd, 2020.</span></div>
     </li>
+
     <li>
       <div class="collapsible-header black clemson-orange-text">Who is eligible for the event?</div>
-      <div class="collapsible-body"><span>Only university and college teams are eligible for CUCTF.</span></div>
+      <div class="collapsible-body"><span>Only university and college teams are eligible for CUCTF 2020.</span></div>
     </li>
     <li>
       <div class="collapsible-header black clemson-orange-text">How many people are allowed on a team?</div>
-      <div class="collapsible-body"><span>You may have up to 4 members on a competing team and, optionally, up to 2 members for the CTF Bootcamp.</span></div>
+      <div class="collapsible-body"><span>Under normal conditions team size is limited to 4 students. However, because CUCTF 2020 is a remote event, team size does not matter for CUCTF 2020. Any amount of students from the same university can join a team. </span></div>
```

Comment:
Agree with Matt E's sentiment of the repetition sounds a little awkward.

Stray space before closing tags

Reply:
Yeah idk what I was thinking there, thanks for the catch.